python/src/IterativeMoments_doc.i.in

```diff
@@ -0,0 +1,115 @@
+
+
+%feature("docstring") OT::IterativeMoments
+"Iterative moments.
+
+Parameters
+----------
+order : int
+    Maximum order of the moments wanted
+
+dimension : int
+    Dimension of the input data
+
+Notes
+-----
+This class iteratively computes the centered moments up to a prescribed order of several :class:`~openturns.Point` or :class:`~openturns.Sample`.
+
+See also
+--------
+IterativeAlgorithm"
+
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::IterativeMoments::getKurtosis
+"Returns the current value of the iterative kurtosis
```

Comment:
There is an inconsistency between this sentence and the one used for `getSkewness` and other get methods: either always write "from internal data" or never. But I think it should be never because "from internal data" seems to contradict statements from class descriptions of iterative algorithms saying no data is stored.

Reply:
OK. Done